pages/case-studies/revio.tsx

```diff
@@ -153,10 +153,10 @@ function RevioDetails() {
       <div className="relative flex -mt-20 h-[1100px] w-full overflow-hidden">
         <div className="absolute bg-primary h-[1100px] w-full overflow-hidden -z-10" />
         <div className="flex w-full mt-60">
-          <div className="w-1/3 pl-64">
+          <div className="w-2/3 pl-64">
```

Comment:
I dont know why this width was changed to 2/3, because now you have 2 divs with width 2/3 which means its 4/3 total width is too much